.gitignore

```diff
@@ -7,6 +7,9 @@
 # dependencies
 /node_modules
 
+# bak
+*.bak
+
```

Comment:
Where do you create `*.bak` files?

Reply:
[here](https://github.com/madoar/angular-archwizard/pull/282/files#diff-379227cf3f95f8b7d73bdce0f45ff974R2)